configurations/leap_s04_pipeline.py

```diff
@@ -0,0 +1,372 @@
+from core_data_modules.cleaners import swahili
+
+from src.pipeline_configuration_spec import *
+
+
+PIPELINE_CONFIGURATION = PipelineConfiguration(
+    pipeline_name="leap_s04",
+    engagement_database=EngagementDatabaseClientConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-engagement-databases-firebase-credentials-file.json",
+        database_path="engagement_databases/POOL-WUSC"
+    ),
+    uuid_table=UUIDTableClientConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+        table_name="avf-global-urn-to-participant-uuid",
+        uuid_prefix="avf-participant-uuid-"
+    ),
+    operations_dashboard=OperationsDashboardConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-dashboards-firebase-adminsdk-gvecb-ef772e79b6.json",
+    ),
+    rapid_pro_sources=[
+        RapidProSource(
+            rapid_pro=RapidProClientConfiguration(
+                domain="textit.com",
+                token_file_url="gs://avf-credentials/wusc-leap-kalobeyei-textit-token.txt"
+            ),
+            sync_config=RapidProToEngagementDBConfiguration(
+                flow_result_configurations=[
+                    FlowResultConfiguration("wusc_leap_s04_kalobeyei_demogs", "preffered_language",
+                                            "kakuma_preffered_language"),
+                    FlowResultConfiguration("wusc_leap_s04_kalobeyei_demogs", "age",
+                                            "kakuma_age"),
+                    FlowResultConfiguration("wusc_leap_s04_kalobeyei_demogs", "location",
+                                            "kakuma_location"),
+                    FlowResultConfiguration("wusc_leap_s04_kalobeyei_demogs", "nationality",
+                                            "kakuma_nationality"),
+                    FlowResultConfiguration("wusc_leap_s04_kalobeyei_demogs", "gender",
+                                            "kakuma_gender"),
+                    FlowResultConfiguration("wusc_leap_s04_kalobeyei_demogs", "disability",
+                                            "kakuma_disabled"),
```

Comment:
Is the kakuma_disabled field new or used before?

(no need to say in the code, just curious here)